app/controllers/reservations_controller.rb

```diff
@@ -2,10 +2,17 @@
 
 class ReservationsController < ApplicationController
   before_action :set_screening, only: %i[new create]
+  before_action :set_reservation, only: %i[edit update destroy]
+  def index
```

Comment:
Missing empty space, rubocop would not be happy :)
```suggestion
  before_action :set_reservation, only: %i[edit update destroy]

  def index
```